site/index.ts

```diff
@@ -159,10 +161,22 @@ domReady(() => {
             },
             imageUpload: imageUploadOptions,
             externalPlugins: [StackSnippetsPlugin],
-        });
+        };
+
+        const editorInstance = new StacksEditor(place, content.value, options);
+
         // set the instance on the window for developers to poke around in
         // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
         (window as any)["editorInstance"] = editorInstance;
+
+        if ( place2 != null)
+        {
+            const secondEditorInstance = new StacksEditor(place2, content.value, options);
+
+            // set the instance on the window for developers to poke around in
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
+            (window as any)["secondEditorInstance"] = secondEditorInstance;
```

Comment:
This looks fine to me. I've set precedence with the absolute hack above 😂.

No changes necessary.